software/nosql/src/main/java/org/apache/brooklyn/entity/nosql/riak/RiakNodeSshDriver.java

```diff
@@ -108,9 +108,10 @@ public void preInstall() {
 
     @Override
     public void install() {
-        if (entity.getConfig(Attributes.DOWNLOAD_URL) != null) {
-            LOG.warn("Ignoring download.url {}, use download.url.rhelcentos or download.url.mac", entity.getConfig(Attributes.DOWNLOAD_URL));
-        }
+        // TODO download URL can be inferred
+//        if (entity.getConfig(Attributes.DOWNLOAD_URL) != null) {
```

Comment:
Under what circumstances would we uncomment this code? Should we just delete it?
Is the TODO saying that we should infer it, or that the warning was pointless because we do infer it?